test/src/unit-QueryCondition.cc

```diff
@@ -719,7 +719,9 @@ void test_apply<char*>(const Datatype type, bool var_size, bool nullable) {
   Range range(bounds, 2 * sizeof(uint32_t));
   REQUIRE(dim.set_domain(range).ok());
   REQUIRE(domain.add_dimension(&dim).ok());
-  REQUIRE(array_schema.set_domain(&domain).ok());
+  REQUIRE(array_schema
+              .set_domain(tdb::make_shared<tiledb::sm::Domain>(HERE(), &domain))
```

Comment:
you may change tdb::make_shared to make_shared after tiledb/common/common.h is included.